<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>NewAddressForm</class>
    <widget class="QDialog" name="NewAddressForm">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>474</width>
                <height>339</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QFrame" name="frameMain">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <widget class="QFrame" name="frameContent">
                        <property name="geometry">
                            <rect>
                                <x>10</x>
                                <y>20</y>
                                <width>431</width>
                                <height>291</height>
                            </rect>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <layout class="QHBoxLayout" name="layoutContent">
                                    <item>
                                        <widget class="QGroupBox" name="groupBox">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred"
                                                    vsizetype="Expanding">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="title">
                                                <string>选项</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="horizontalLayout">
                                    <property name="spacing">
                                        <number>9</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>9</number>
                                    </property>
                                    <item>
                                        <spacer name="horizontalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>0</width>
                                                    <height>0</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="ButtonOK">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>96</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>96</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>确定</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="ButtonCancel">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="minimumSize">
                                                <size>
                                                    <width>96</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>96</width>
                                                    <height>32</height>
                                                </size>
                                            </property>
                                            <property name="text">
                                                <string>取消</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>
